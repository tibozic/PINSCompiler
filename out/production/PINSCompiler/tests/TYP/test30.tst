typ t1: t2;
typ t2: t3;
typ t3: integer
