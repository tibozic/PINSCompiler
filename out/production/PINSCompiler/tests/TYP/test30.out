Defs [1:1-3:16]
  TypeDef [1:1-1:11]: t1
    # typed as: int
    TypeName [1:9-1:11]: t2
      # defined at: [2:1-2:11]
      # typed as: int
  TypeDef [2:1-2:11]: t2
    # typed as: int
    TypeName [2:9-2:11]: t3
      # defined at: [3:1-3:16]
      # typed as: int
  TypeDef [3:1-3:16]: t3
    # typed as: int
    Atom [3:9-3:16]: INT
      # typed as: int
